apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/VilkaarsvurderingService.kt

```diff
@@ -190,6 +181,37 @@ class VilkaarsvurderingService(
             }
         }
 
+    fun slettVilkaarsvurdering(
```

Comment:
Hvorfor bruker vi `runBlocking` her?
```suggestion
    suspend fun slettVilkaarsvurdering(
        behandlingId: UUID,
        brukerTokenInfo: BrukerTokenInfo,
    ) = vilkaarsvurderingRepository.slettVilkaarvurdering(behandlingId).also {
        behandlingKlient.settBehandlingStatusOpprettet(behandlingId, brukerTokenInfo, true)
    }
```

Reply:
Nitpick, also er ikke nødvendig
```suggestion
    suspend fun slettVilkaarsvurdering(
        behandlingId: UUID,
        brukerTokenInfo: BrukerTokenInfo,
    ) {
        vilkaarsvurderingRepository.slettVilkaarvurdering(behandlingId)
        behandlingKlient.settBehandlingStatusOpprettet(behandlingId, brukerTokenInfo, true)
    }
```

Reply:
[6ac27e4](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2473/commits/6ac27e403cd5a4b96c4c12d344cc5c75cc664930)